src/graphic_model/lomse_box_slice_instr.cpp

```diff
@@ -169,14 +169,19 @@ void GmoBoxSliceStaff::reposition_shapes(const vector<LUnits>& yShifts,
             else if ((*it)->is_shape_note())
             {
                 GmoShapeNote* pShapeNote = static_cast<GmoShapeNote*>(*it);
-                LUnits increment = (yShift + yShifts[m_idxStaff-1]);
```

Comment:
Just by curiosity, how did you spot this bug? An issue in any sample score?

Reply:
This issue affects arpeggio at cross-staff chords, as I have reused this `increment` value for increasing arpeggio length at a staff distance change. Previously this value has only been used for expanding a stem segment between staves, and the extra length resulting from the `+` sign didn't result in any visible artifacts: the elongated stem segment just got drawn on top of another stem segment which doesn't produce any visible difference. With cross-staff arpeggio this results in an incorrect arpeggio length so I had to correct this expression to fix it.